cpp/src/arrow/engine/substrait/test_plan_builder.cc

```diff
@@ -88,16 +91,18 @@ Result<std::unique_ptr<substrait::ProjectRel>> CreateProject(
       // If it doesn't have a type then it's an enum
       const std::string& enum_value = arguments[arg_index];
       auto enum_ = std::make_unique<substrait::FunctionArgument::Enum>();
-      if (enum_value.size() > 0) {
-        enum_->set_specified(enum_value);
-      } else {
-        auto unspecified = std::make_unique<google::protobuf::Empty>();
-        enum_->set_allocated_unspecified(unspecified.release());
-      }
+      enum_->set_specified(enum_value);
```

Comment:
This change seems unintended.

Reply:
It's part of adapting to the new options format.  Function arguments can no longer be unspecified.